README.md

```diff
@@ -1,3 +1,71 @@
-# Encryptonize Go Client Library v3.2.0
+# Go Client Library for Encryptonize&reg;
 
-A simple Encryptonize go client library.
+Go client libraries for
+* [Encryptonize&reg; Core](https://github.com/cyber-crypt-com/encryptonize-core)
+* [Encryptonize&reg; Objects](https://github.com/cyber-crypt-com/encryptonize-objects)
+* [Encryptonize&reg; Key Server](TODO)
+
+## Usage
+The easiest way to use the Encryptonize&reg; Core client is through the `NewCoreClientWR`
+constructor. The resulting client will automatically refresh the user's access token when it
+expires. For details on how to create a user, see the Encryptonize&reg; Core [user
+manual](https://github.com/cyber-crypt-com/encryptonize-core/blob/master/documentation/user_manual.md).
+
+```go
```

Comment:
Shouldn't we just have a link to the godoc with the usage example code?
Seems like we would have to maintain the same code in two different locations this way.

Reply:
I think it's good to have some code on this page to catch people's attention. The godocs hold more examples for those that are interested. If there was some way of embedding it, that'd be great.